GitUI/SplitterManager.cs

```diff
@@ -0,0 +1,192 @@
+﻿using GitUIPluginInterfaces;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public class SplitterManager
+    {
+        private readonly ISettingsSource _settings;
+        private readonly List<SplitterData> splitters = new List<SplitterData>();
+        private float _designTimeFontSize;
+
+        public SplitterManager(ISettingsSource settings, float designTimeFontSize = 8.25F)
```

Comment:
I've been experimenting, while dealing with scaling of the dashboard controls, and it seems the following code is the best way to get the scale factor: 

```
var currentDpi = GetCurrentDeviceDpi();
var standardDpi = 96; // DPI at 100%
var scaleFactor = (float)currentDpi / standardDpi;
```

100% or scale factor 1.0
<img width="471" alt="screen shot 2017-05-14 at 2 47 24 pm" src="https://cloud.githubusercontent.com/assets/4403806/26031475/45dac540-38b8-11e7-92ca-4edc5791bd5f.png">

125%
<img width="625" alt="screen shot 2017-05-14 at 2 49 26 pm" src="https://cloud.githubusercontent.com/assets/4403806/26031473/40998af8-38b8-11e7-8b7a-3c5d2760659a.png">

150%
<img width="714" alt="screen shot 2017-05-14 at 2 50 29 pm" src="https://cloud.githubusercontent.com/assets/4403806/26031474/42585dc4-38b8-11e7-8abe-020de855b9f5.png">